backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -145,7 +145,10 @@ protected Specification<Person> buildPersonSearchFilter(
             : Arrays.stream(namePrefixMatch.split("[ ,]")).collect(Collectors.toList());
 
     // build up filter based on params
-    Specification<Person> filter = inCurrentOrganizationFilter().and(isDeletedFilter(isArchived));
+    Specification<Person> filter = inCurrentOrganizationFilter();
+    if (!isArchived) {
```

Comment:
non-blocking personal opinion nit: this parameter behaves more like an `includeArchived` in this method as well 😅

Reply:
Very fair point, done in [990411b](https://github.com/CDCgov/prime-simplereport/pull/4229/commits/990411be2ee1fd3f44795b114f5b67518b14e7f8)